archetypes/Exchange/Inputs/index.tsx

```diff
@@ -43,6 +53,16 @@ export default (({ pool, userBalances, swapState, swapDispatch }) => {
 
     const isLong = side === SideEnum.long;
     const token = useMemo(() => (isLong ? pool.longToken : pool.shortToken), [isLong, pool.longToken, pool.shortToken]);
+
+    const escrowTokenBalance = useMemo(() => {
+        switch (balanceType) {
+            case BalanceTypeEnum.wallet:
+                return isLong ? userBalances.aggregateBalances.longTokens : userBalances.aggregateBalances.shortTokens;
+            default:
+                return isLong ? userBalances.longToken.balance : userBalances.shortToken.balance;
+        }
```

Comment:
this variable is called `escrowTokenBalance` but sometimes its actually the wallet balance. this is kinda confusing and if we need logic to use escrow sometimes and wallet other times it should be in the rendering of the component imo. have a feeling its the cause of this unexpected behaviour:

![image](https://user-images.githubusercontent.com/14157626/178166382-b5accdb7-6b19-4114-8d2b-db10d4618d3e.png)

![image](https://user-images.githubusercontent.com/14157626/178166392-395483e2-7690-48e5-be4b-7e77e48b0c62.png)

notice it says the full available amount when I'm committing from wallet and then specifically highlights escrow balance when I switch to committing from escrow